app/src/main/java/com/premar/muvi/rest/ApiService.java

```diff
@@ -29,13 +30,18 @@
             @Query("api_key") String apiKey,
             @Query("sort_by") String sort_by);
 
-    @GET("/movie/{id}")
-    Call<Movie> getMovie(@Path("id") int id,
+    @GET("movie/{movie_id}")
+    Call<Movie> getMovie(@Path("movie_id") int id,
                          @Query("api_key") String apiKey);
 
-    @GET("/movie/{movie_id}/videos")
-    Call<TrailerResponse> getMovieTrailers(@Path("movie_id") int id,
-                                           @Query("api_key") String apiKey);
+    @GET("movie/{movie_id}/videos")
```

Comment:
###Please I need a hand on this, am trying to fetch data to populate the Trailer recyclerview, it seems it is null